calling/src/androidMain/kotlin/com/wire/kalium/calling/Calling.kt

```diff
@@ -22,6 +22,17 @@ import com.wire.kalium.calling.types.Uint32_t
 // A magic number used to initialize AVS (required for all mobile platforms).
 const val ENVIRONMENT_DEFAULT = 0
 
+/**
+ * WCALL_CALL_TYPE_NORMAL          0
+ * WCALL_CALL_TYPE_VIDEO           1
+ * WCALL_CALL_TYPE_FORCED_AUDIO    2
+ */
+object CallType {
```

Comment:
```suggestion
internal object CallType {
```

If this is only used in `calling`, we can hide it.

Reply:
It is not actually, it is also being used in `CallManger` in the android package, where it accepts an incoming call with its type, can [see here](https://github.com/wireapp/kalium/blob/d55c4ddb3db8898ab3ff21329f4cb09a25261496/logic/src/androidMain/kotlin/com/wire/kalium/logic/feature/call/CallManager.kt) on line 205.